db/migrate/20191101184930_create_vendors.rb

```diff
@@ -0,0 +1,10 @@
+class CreateVendors < ActiveRecord::Migration[5.1]
```

Comment:
migration to add a new vendors table to the db. This will result in a change in schema as well.